tests/test_scene_rendering/simple_scenes.py

```diff
@@ -34,3 +34,9 @@ def construct(self):
         dot = Dot().set_color(GREEN)
         self.add(dot)
         self.wait(1)
+
+
+class Text(Scene):
```

Comment:
Your declaration of a class Text conflicts with the imported Text class. Name it something like NoAnimationsText to avoid the issue.